bin/config.js

```diff
@@ -1,19 +1,31 @@
-const rc = require('rc')
+const { writeFile } = require('fs')
+const configDefaults = require('./files/configDefaults.json')
+const minimist = require('minimist')
 
-module.exports = rc('cordlr', {
-  plugins: [],
-  prefix: '$',
-  adminPerms: [
-    'KICK_MEMBERS',
-    'BAN_MEMBERS',
-    'ADMINISTRATOR',
-    'MANAGE_CHANNELS',
-    'MANAGE_GUILD',
-    'MANAGE_MESSAGES',
-    'MUTE_MEMBERS',
-    'DEAFEN_MEMBERS',
-    'MOVE_MEMBERS',
-    'MANAGE_NICKNAMES',
-    'MANAGE_ROLES_OR_PERMISSIONS'
-  ]
-})
+const configPath = `${process.cwd()}/package.json`
+let config
+try {
+  config = require(configPath)
+} catch (e) {
+  // file doesn't exist
+  config = { cordlr: configDefaults }
+}
+
+// insert command line options
+const args = minimist(process.argv.slice(2))
+for (const key of Object.keys(args)) {
```

Comment:
Hmm,instead of looping, could we use `Object.assign` maybe?:

```js
let config = minimist(process.argv.slice(2), {
  // ...
})

try {
  Object.assign(config, require(configPath))
} catch (e) {  }

// ...
```